KraflaPartialLibrary.aml

```diff
@@ -0,0 +1,193 @@
+          <Value>Out</Value>
+          <Constraint Name="LegalValues">
+            <NominalScaledType>
+              <RequiredValue>Out</RequiredValue>
+            </NominalScaledType>
+          </Constraint>
+        </Attribute>
+      </InterfaceClass>
+    </InterfaceClass>
+  </InterfaceClassLib>
+  <SystemUnitClassLib Name="StructureClassLibrary">
+    <SystemUnitClass Name="ContextHidingStructure" ID="f3fd310e-755d-40e0-91a3-bc09e2492b11">
+      <Description>Structure for Context and Hiding</Description>
+      <SupportedRoleClass RefRoleClassPath="AutomationMLBaseRoleClassLib/AutomationMLBaseRole/Structure" />
+    </SystemUnitClass>
+  </SystemUnitClassLib>
+  <SystemUnitClassLib Name="DocumentClassLibrary">
+    <SystemUnitClass Name="ContextHiding" ID="6c71de7c-daad-4a4b-9a30-aa86d48049a8">
+      <Description>Context and Hiding document</Description>
+    </SystemUnitClass>
+    <SystemUnitClass Name="ShutdownHierarchy" ID="19c83e4c-a700-4683-baed-49488b5caf5d">
+      <Description>Shutdown Hierarchy</Description>
+      <Attribute Name="DrawingSize" AttributeDataType="xs:string">
+        <Description>Drawing size, restricted to A1 ref ISO 5457</Description>
+        <DefaultValue>A1</DefaultValue>
+        <Value />
+        <Constraint Name="Requirement">
+          <NominalScaledType>
+            <RequiredValue>A1</RequiredValue>
+          </NominalScaledType>
+        </Constraint>
+      </Attribute>
+      <Attribute Name="Orientation" AttributeDataType="xs:string">
+        <Description>Restricted to Landscape</Description>
+        <DefaultValue>Landscape</DefaultValue>
+        <Value />
+        <Constraint Name="Requirement">
+          <NominalScaledType>
+            <RequiredValue>Landscape</RequiredValue>
+          </NominalScaledType>
+        </Constraint>
+      </Attribute>
+      <Attribute Name="CoordinateSystem" AttributeDataType="xs:string">
+        <Description>https://en.wikipedia.org/wiki/Cartesian_coordinate_system</Description>
+        <DefaultValue />
+        <Value />
+        <Constraint Name="Requirement">
+          <NominalScaledType>
+            <RequiredValue>right-handed</RequiredValue>
+          </NominalScaledType>
+        </Constraint>
+      </Attribute>
+      <Attribute Name="DrawingOrigin" AttributeDataType="xs:string">
+        <Description />
+        <DefaultValue />
+        <Value />
+        <Constraint Name="Requirement">
+          <NominalScaledType>
+            <RequiredValue>Lower left corner</RequiredValue>
+          </NominalScaledType>
+        </Constraint>
+      </Attribute>
+      <Attribute Name="Scale" AttributeDataType="xs:string">
+        <Description />
+        <DefaultValue />
+        <Value />
+      </Attribute>
+    </SystemUnitClass>
+  </SystemUnitClassLib>
+  <SystemUnitClassLib Name="ContextHidingElementLibrary">
+    <Version>0</Version>
+    <SystemUnitClass Name="Transition" ID="1e580c24-ffc9-4399-b5c7-b226dcfe385b">
```

Comment:
Should there be a RefBaseClassPath on 'Transition'?

Reply:
There is no parent class for Context and Transition classes. Same as for example classes in DocumentElementClassLibrary.